cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -74,15 +74,14 @@ def configure_websocket():
     Currently the fixture cfme/fixtures/base.py,
     disables the websocket role to avoid intrusive popups.
     """
-    roles = configuration.get_server_roles()
-    if 'websocket' in roles and not roles['websocket']:
-        logger.info('Enabling the websocket role to allow console connections')
-        roles['websocket'] = True
-        configuration.set_server_roles(**roles)
-        yield
+    roles = dict()
+    logger.info('Enabling the websocket role to allow console connections')
+    roles['websocket'] = True
+    configuration.set_server_roles(db=False, **roles)
```

Comment:
What's the reason to set the roles via UI?

Reply:
Use context manager in that case like here https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/control/test_alerts.py#L131

Reply:
Ok will give it a try.